langchain-core/src/runnables/base.ts

```diff
@@ -395,8 +398,13 @@ export abstract class Runnable<
     delete config.runId;
     let output;
     try {
+      // To support OrchestratorAbortBehavior.COMPLETE_PENDING, we need to check the signal here and
+      // throw if it aborted rather than starting additional tasks
+      if (checkAbortSignal(config)) {
+        config?.signal?.throwIfAborted();
+      }
```

Comment:
Bit of a self nit, but I don't love this pattern for this utility function. The `checkAbortSignal` function itself will throw on abort if `throw_immediately` is set (default). Maybe it should have a required flag that tells it if there are pending active tasks?